cluster/src/test/java/com/hartwig/pipeline/tertiary/orange/OrangeTest.java

```diff
@@ -99,7 +100,7 @@ protected List<String> expectedCommands() {
                         + "-cohort_percentiles_tsv /opt/resources/orange/cohort_percentiles.tsv "
                         + "-driver_gene_panel_tsv /opt/resources/gene_panel/37/DriverGenePanel.37.tsv "
                         + "-known_fusion_file /opt/resources/fusions/37/known_fusion_data.37.csv "
-                        + "-ensembl_data_directory /opt/resources/ensembl_data_cache/37/ " + "-convert_germline_to_somatic";
+                        + "-ensembl_data_directory /opt/resources/ensembl_data_cache/37/ -experiment_date 2023-05-19";
```

Comment:
This experiment date will give a WARN in ORANGE, since only supported format is yyMMdd.

Reply:
Ok, it would be best if we stuck with ISO convention date formats but probably not worth an ORANGE release. I'll update.

Reply:
It would be nice if you could update ORANGE nonetheless I think, just don't make a release yet. 

In addition to supporting ISO, I think it would be better to throw an exception if someone explicitly passed a date in a format that is not supported (reverting to `now` feels arbitrary)

```
    @NotNull
    private static LocalDate interpretExperimentDateParam(@NotNull String experimentDateString) {
        String format = "yyMMdd";

        LocalDate experimentDate;
        try {
            experimentDate = LocalDate.parse(experimentDateString, DateTimeFormatter.ofPattern(format, Locale.ENGLISH));
            LOGGER.debug("Configured experiment date to {}", experimentDate);
        } catch (DateTimeParseException exception) {
            experimentDate = LocalDate.now();
            LOGGER.warn("Could not parse configured experiment date '{}'. Expected format is '{}'", experimentDateString, format);
        }
        return experimentDate;
    }
```